src/TestSuite/Stub/ConsoleOutput.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * CakePHP(tm) Tests <http://book.cakephp.org/2.0/en/development/testing.html>
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @since         3.0.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+
+namespace Cake\TestSuite\Stub;
+
+use Cake\Console\ConsoleOutput as Base;
+
+/**
+ * A ConsoleOutput class intended for test cases.
+ */
+class ConsoleOutput extends Base
```

Comment:
I recently put something very similar into the 3.1 branch. It works a bit differently, and it would be great if we could have the same implementation.  [3.1 approach](https://github.com/cakephp/cakephp/blob/3.1/src/TestSuite/Stub/ConsoleOutput.php).